projects/packages/publicize/src/class-publicize-ui.php

```diff
@@ -63,6 +63,8 @@ public function init() {
 
 	/**
 	 * If the ShareDaddy plugin is not active we need to add the sharing settings page to the menu still
+	 *
+	 * @deprecated
```

Comment:
I would recommend adding the version, here and elsewhere. It makes it easier for everyone to know when it was deprecated, when it can be removed.

```suggestion
	 * @deprecated $$next-version$$
```

Reply:
TIL. Updated!